vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -858,6 +858,20 @@ testContainerImagePrefetchScript() {
   return 0
 }
 
+testBccTools () {
+  result=0 
+  for line in '- bcc-tools' '- libbcc-examples'; do
+    if ! grep -F -x -e "$line" /opt/azure/vhd-install.complete; then
+      echo "$line is missing"
+      result=1
+    fi
+  done
```

Comment:
Can be simplified - just return 1 inside the `then` condition if line is missing